commands/user.go

```diff
@@ -169,6 +171,44 @@ var UserConvertCmd = &cobra.Command{
 	Args: cobra.MinimumNArgs(1),
 }
 
+var MigrateAuthCmd = &cobra.Command{
+	Use:     "migrate_auth [from_auth] [to_auth] [migration-options]",
+	Short:   "Mass migrate user accounts authentication type",
+	Long:    `Migrates accounts from one authentication provider to another. For example, you can upgrade your authentication provider from email to ldap.`,
+	Example: "user migrate_auth email saml users.json",
+	Args: func(command *cobra.Command, args []string) error {
+		if len(args) < 2 {
+			return errors.New("auth migration requires at least 2 arguments")
+		}
+
+		toAuth := args[1]
+
+		if toAuth != "ldap" && toAuth != "saml" { // nolint: goconst
```

Comment:
Why not just make them const?

Reply:
0/5 on this actually. I just didn't want to increase verbosity.

Reply:
Although I agree on this creating verbosity, I think it's worth to either follow the linter or tweak it to avoid these cases if it make sense

Reply:
I prefer to tweak linter :)